scripts/build/linux-cross

```diff
@@ -13,6 +13,7 @@ CROSS_OSARCH="linux/amd64 linux/arm linux/ppc64le"
 # linux/s390x
 
 echo "Building all linux binaries"
+# TODO(stevvooe): Remove gox. This just wraps environement variables.
```

Comment:
I think we're talking about different things.

`cross` is not "build for x". It is a target for "build everything".

You can already "build for x" by using `script/build/binary` with any environment variables you want.